### PR TITLE
ENDOC-6 Add containers/repo summary and expand on backup/restore tuto…

### PR DESCRIPTION
**Pull request by nshaw**
…rial.

This branch is currently deployed to the staging site at https://es-entando.github.io/docs/reference/deployment-structure.html

### CODE REVIEW

**Comment by nshaw** [comment]
@joewhite101 , this is almost verbatim your content, with formatting added. I expanded the backup/restore tutorial per your notes.

File: vuepress/docs/docs/reference/deployment-structure.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+functionality in versions of Entando prior to Entando 6. In a quickstart deployment the _entando-de-app_ is deployed as part of the _entando-composite-app_ multi 
+container pod.
+* GitHub: <https://github.com/entando-k8s/entando-de-app/>
+* DockerHub: <https://hub.docker.com/repository/docker/entando/entando-de-app-eap>,<https://hub.docker.com/repository/docker/entando/entando-de-app-wildfly>
+
+#### Customization
+The _entando-de-app_ is very likely to be customized as part of an Entando implementation. This image can be 
+customized with new APIs, legacy Entando plugins, new database tables, or other extensions to the _entando-core_. 
+It is highly recommended that most extensions to the platform in Entando 6 occur in microservices, 
+however for legacy integrations, extensions to the CMS, or migrations from earlier Entando versions many applications will make
```
Comment (joewhite101) [inline vuepress/docs/docs/reference/deployment-structure.md:62]: Typo or wrong word many --> may

File: vuepress/docs/docs/reference/deployment-structure.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+deployment. In a quickstart deployment the _app-builder_ container is deployed in the _entando-composite-app_ 
+multiple container pod. The _app-builder_ also communicates with the Component Manager via REST API to fetch 
+information about Entando Bundles deployed to the Entando Component Repository (ECR).
+
+* GitHub: <https://github.com/entando/app-builder/>
+* DockerHub: <https://hub.docker.com/repository/docker/entando/app-builder/>
+
+#### Customization
+The _app-builder_ is built to be customized and will be customized as part of many Entando implementations. 
+The _app-builder_ can be customized at runtime via micro frontend
```
Comment (joewhite101) [inline vuepress/docs/docs/reference/deployment-structure.md:77]: missing "s" on frontend

File: vuepress/docs/tutorials/devops/backing-up-and-restoring-your-environment.md [hunk 3]
```diff
@@ -11,38 +11,55 @@
...
-        them the assets and DB an image and
+        the static assets and database backup into a Docker image.
 
     -   Entando will automatically instantiate and populate the app from
         the most recent backup
 
-4.  Move the resources and protected folders from the zip file into your
-    `entando-de-app` in `src/main/webapp` replacing any content that is
-    already there
+4.  Move the resources and protected folders from your Entando backup (see #TODO) into your
```
Comment (joewhite101) [inline vuepress/docs/tutorials/devops/backing-up-and-restoring-your-environment.md:60]: TODO here

File: vuepress/docs/docs/reference/deployment-structure.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
+# Entando Deployment Structure
+
+This page provides a high level overview of the key Entando github repositories along with a brief description
```
Comment (joewhite101) [inline vuepress/docs/docs/reference/deployment-structure.md:3]: github --> GitHub

File: vuepress/docs/docs/reference/deployment-structure.md (hunk 1 above)
Comment (nshaw) [reply vuepress/docs/docs/reference/deployment-structure.md]: Reworded a bit.

File: vuepress/docs/docs/reference/deployment-structure.md (hunk 2 above)
Comment (nshaw) [reply vuepress/docs/docs/reference/deployment-structure.md]: Updated

File: vuepress/docs/tutorials/devops/backing-up-and-restoring-your-environment.md (hunk 3 above)
Comment (nshaw) [reply vuepress/docs/tutorials/devops/backing-up-and-restoring-your-environment.md]: Good catch, fixed now.

File: vuepress/docs/docs/reference/deployment-structure.md (hunk 4 above)
Comment (nshaw) [reply vuepress/docs/docs/reference/deployment-structure.md]: Fixed

**Pull request merged**